extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/conf/JacksonLDAPConfiguration.java

```diff
@@ -203,6 +203,13 @@ public class JacksonLDAPConfiguration implements LDAPConfiguration {
      */
     @JsonProperty("member-attribute-type")
     private String memberAttributeType;
+    
+    /**
+     * The raw YAML value of {@link LDAPGuacamoleProperties#LDAP_USERNAMES_CASE_SENSITIVE}.
+     * If not set within the YAML, this will currently default to true.
+     */
+    @JsonProperty("ldap-usernames-case-sensitive")
+    private String caseSensitiveUsernames;
```

Comment:
These values do follow the names of the properties, but:

* Since the values within `ldap-servers.yaml` are always LDAP-related, the `ldap-` prefix used by the Guacamole properties is omitted.
* The Guacamole property that `LDAP_USERNAMES_CASE_SENSITIVE` points to is actually `ldap-case-sensitive-usernames`, not `ldap-usernames-case-sensitive`.

Since all the other properties use the `*-case-sensitive-usernames` convention, as well, I think:

* This should be renamed to `@JsonProperty("case-sensitive-usernames")`.
* The constant in `LDAPGuacamoleProperties` defining the property should be renamed to `LDAP_CASE_SENSITIVE_USERNAMES`.